misc/python/materialize/feature_benchmark/benchmark.py

```diff
@@ -83,6 +83,7 @@ def run(self) -> List[Aggregation]:
             print("init() done")
 
         for i in range(sys.maxsize):
+            # TODO: Philip: When does scenario.before() return a list? Why?
```

Comment:
@philip-stoev?

Reply:
You are allowed to have multiple separate actions in the `before()` block of a scenario, even though currently no scenario is using this ability.

Reply:
Any reason against changing the return type to list?

Reply:
✅